src/fairseq2/metrics/recorders/composite.py

```diff
@@ -15,17 +15,27 @@
 
 @final
 class CompositeMetricRecorder(MetricRecorder):
+    """
+    Represents a collection defining a recorder for multiple metrics
+    """
     def __init__(self, recorders: Collection[MetricRecorder]) -> None:
         self._recorders = recorders
 
     @override
     def record_metric_values(
         self, category: str, values: Mapping[str, object], step_nr: int | None = None
     ) -> None:
+        """
```

Comment:
I don't think we need to add documentation for overridden methods. The docstr of the method on the corresponding interface (in this case `MetricRecorder`) should be descriptive enough to explain the expected behavior of the metod.